### ISSUE TITLE
Couldn't find CONTRIBUTORS.txt

### ISSUE THREAD

**Issue by sks444** [pythonindia/junction#635]
https://github.com/pythonindia/junction/blob/master/.github/CONTRIBUTING.rst#pull-requests

Mentions that you should add yourself to `CONTRIBUTORS.txt` if you're not there. But I couldn't find the file. Am I missing something?

I think we are directly adding the contributor's GitHub profile to README.md instead of putting it in `CONTRIBUTORS.txt`.

**Comment by ananyo2012** [comment]
I use @allcontributors for adding contributors in readme. Simplified the process. If we are favoring to use that we may need to keep the contributors list in README.

**Comment by pradyunsg** [comment]
Honestly, I prefer removing the list from the README. If we want, we can automagically generate the list and have it in the documentation.

I'll be revamping the README later today; once our documentation pipeline is set up.

**Comment by palnabarun** [comment]
Ref: https://github.com/pythonindia/pyconindia-handbook/pull/25 for the discussion on having contributors list in README.

---
TL;DR It is not favorable to keep the list of contributors in README.

**Comment by palnabarun** [comment]
Also, since https://github.com/pythonindia/junction/blob/master/.github/CONTRIBUTING.rst#pull-requests doesn't mention adding one's name to CONTRIBUTORS.txt, this issue is redundant as such. Hence, closing.
